### ISSUE TITLE
我在复现您的代码的时候,出现了下面的问题

### ISSUE THREAD

**Issue by 15755841658** [idiap/CNN_QbE_STD#3]
(kws-zy) cad-1@cad1-SYS-7048GR-TR:/media/cad-1/A42A1B372A1B0648/CNN_QbE_STD-master$ python query_detection_dtw_cnn.py -optim adam -learning_rate 0.0001 -input_size 152 -batch_size 50 -layers 9 -depth 30 -dropout 0.2 -loss_threshold 0.1 -n_valid 50 -max_batch_dev 250 -max_batch_train 1000
usage: query_detection_dtw_cnn.py [-h] [-depth DEPTH] [-input_size INPUT_SIZE]
                                  [-load_model] [-modelpath MODELPATH]
                                  [-batch_size BATCH_SIZE] [-epochs EPOCHS]
                                  [-start_epoch START_EPOCH] [-optim OPTIM]
                                  [-learning_rate LEARNING_RATE] [-loss LOSS]
                                  [-dropout DROPOUT]
                                  [-max_batch_dev MAX_BATCH_DEV]
                                  [-max_batch_train MAX_BATCH_TRAIN]
                                  [-shuffle] [-gpus] [-datapath DATAPATH]
                                  [-outpath OUTPATH] [-n_valid N_VALID]
                                  [-mfcc] [-loss_threshold LOSS_THRESHOLD]
query_detection_dtw_cnn.py: error: unrecognized arguments: -layers 9
您的代码里面好像没有设置参数-layers

**Comment by 595145489** [comment]
你好,请问要与我交流下吗,我的账号上有QQ或者email,这里提的issue好像官方不是很会回的样子 @15755841658

**Comment by dhananjay-ram** [comment]
Hi, 
I noticed that argument was a mistake. I have changed in the README file. Please check and let me know if there are any other issues.
@15755841658
